demisto_sdk/commands/common/tests/pack_unique_files_test.py

```diff
@@ -259,6 +259,36 @@ def test_is_approved_tags(self, repo, tags, is_valid):
         if not is_valid:
             assert 'The pack metadata contains non approved tags: NonApprovedTag' in self.validator.get_errors()
 
+    @pytest.mark.parametrize('pack_content, tags, is_valid', [
+        ("none", [], True),
+        ("none", ["Use Case"], False),
+        ("playbook", ["Use Case"], True),
+        ("incident", ["Use Case"], True),
+        ("layout", ["Use Case"], True),
+        ("playbook", [], True),
+    ])
+    def test_is_right_usage_of_usecase_tag(self, repo, pack_content, tags, is_valid):
+        pack_name = 'PackName'
+        pack = repo.create_pack(pack_name)
+        pack.pack_metadata.write_json({
+            PACK_METADATA_USE_CASES: [],
+            PACK_METADATA_SUPPORT: XSOAR_SUPPORT,
+            PACK_METADATA_TAGS: tags
+        })
+
+        if pack_content == "playbook":
+            playbook = pack.create_playbook(name="PlaybookName")
+            pack.playbooks.append(playbook)
```

Comment:
redundant, the `create_playbook` function already append it (same for the others).

Reply:
i did it first without and the test failed, while debugging it showed that it doesn't have any playbooks

Reply:
this sounds weird, can you please double-check (add a screenshot)

Reply:
i did double check and it worked, weird, just fixed it.